coins.json

```diff
@@ -602,5 +602,17 @@
         "publicKeyType": "ed25519",
         "transactionPath": "/transaction/",
         "explorer": "https://semux.info/explorer"
+    },
+    {
+        "id": "aeternity",
+        "name": "Aeternity",
+        "symbol": "AE",
+        "decimals": 18,
+        "blockchain": "Aeternity",
+        "derivationPath": "m/44'/0'/0'/0",
```

Comment:
coinType should be `457` https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
true that